### PR TITLE
Update sig scalability readme

### PR DESCRIPTION
**Pull request by wojtek-t**
/assign @shyamjvs 
@thockin @kubernetes/sig-scalability-misc

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -1611,13 +1616,19 @@ sigs:
...
+    - name: kubernetes-scalability-definition
       owners:
-      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/cmd/kubemark/OWNERS
-      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/kubemark/OWNERS
-    - name: perf-tests
+      - https://github.com/kubernetes/community/blob/master/sig-scalability/slos/OWNERS
+    - name: scalability-governance
+      owners:
+      - https://github.com/kubernetes/community/blob/master/sig-scalability/governance/OWNERS
+    - name: scalability-governance
```
Comment (shyamjvs) [inline sigs.yaml]: This is repeated twice?

File: sigs.yaml [hunk 2]
```diff
@@ -1611,13 +1616,19 @@ sigs:
...
       owners:
-      - https://raw.githubusercontent.com/kubernetes/perf-tests/master/OWNERS
+      - https://github.com/kubernetes/community/blob/master/sig-scalability/governance/OWNERS
+    - name: scalability-test-frameworks
+      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/kubemark/OWNERS
+      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/test/kubemark/OWNERS
+      - https://raw.githubusercontent.com/kubernetes/perf-tests/blob/master/clusterloader2/OWNERS
```
Comment (shyamjvs) [inline sigs.yaml]: Aren't we the owners for perf-tests root? Maybe except something around networking (which we may also assume)?

File: sigs.yaml [hunk 3]
```diff
@@ -1611,13 +1616,19 @@ sigs:
       - name: sig-scalability-test-failures
         description: Test Failures and Triage
     subprojects:
-    - name: kubemark
+    - name: kubernetes-scalability-definition
```
Comment (shyamjvs) [inline sigs.yaml]: What about other things under sig-scalability/?
Like configs&limits, processes, etc?

File: sigs.yaml [hunk 4]
```diff
@@ -1564,16 +1564,21 @@ sigs:
...
-      What size clusters do we think that we should support with Kubernetes
-      in the short to medium term? How performant do we think that the control
-      system should be at scale? What resource overhead should the Kubernetes
-      control system reasonably consume?
-
-      For more details about our objectives please review our
-      [Scaling And Performance Goals](https://git.k8s.io/community/sig-scalability/goals.md)
-    charter_link:
+      We are actively working on finding and removing various scalability
+      bottlenecks which should leads us towards pushing system's scalability
```
Comment (shyamjvs) [inline sigs.yaml]: leads -> lead

File: sigs.yaml [hunk 5]
```diff
@@ -1564,16 +1564,21 @@ sigs:
...
+      higher. This may include going beyond 5k nodes in the future - although
+      that's not our priority as of now, this is very deeply in our area of
+      interest and we are happy to guide and collaborate on any efforts towards
+      that goal as long as they are not sacrificing on overal Kubernetes
```
Comment (shyamjvs) [inline sigs.yaml]: overall

File: sigs.yaml [hunk 6]
```diff
@@ -1564,16 +1564,21 @@ sigs:
   - name: Scalability
     dir: sig-scalability
     mission_statement: >
-      Responsible for answering scalability related questions such as:
+      SIG Scalability is responsible for defining and driving scalability goals
+      for Kubernetes. We also coordinate and contribute to general system-wide
+      scalability and performance improvements (not falling into the charter of
+      another individual SIG) by driving large architectural changes and finding
```
Comment (shyamjvs) [inline sigs.yaml]: Maybe "other individual SIGs"?

**Comment by thockin** [comment]
Thanks!  This messages aligns with my thinking very well.

File: sigs.yaml (hunk 6 above)
Comment (wojtek-t) [reply sigs.yaml]: done

File: sigs.yaml (hunk 4 above)
Comment (wojtek-t) [reply sigs.yaml]: done

**Comment by shyamjvs** [comment]
+1, well-written!

File: sigs.yaml (hunk 5 above)
Comment (wojtek-t) [reply sigs.yaml]: done

File: sigs.yaml (hunk 1 above)
Comment (wojtek-t) [reply sigs.yaml]: done

File: sigs.yaml (hunk 2 above)
Comment (wojtek-t) [reply sigs.yaml]: We are. I wanted to mention clusterloader2 explicitly. But I'm happy to add perf-tests/owners too here.

File: sigs.yaml (hunk 3 above)
Comment (wojtek-t) [reply sigs.yaml]: That was supposed to be very initial version as I don't have good name for it.
I would like to extend it a bit more (in particular add more descriptions etc., which I wrote some time ago when I was working on charter (but then the template for charter changed).
But i wanted to push have it in sooner.

Does that sound ok for you?

File: sigs.yaml [hunk 7]
```diff
@@ -1611,13 +1616,18 @@ sigs:
...
-    - name: perf-tests
+      - https://github.com/kubernetes/community/blob/master/sig-scalability/slos/OWNERS
+    - name: kubernetes-scalability-governance
+      owners:
+      - https://github.com/kubernetes/community/blob/master/sig-scalability/governance/OWNERS
+    - name: kubernetes-scalability-test-frameworks
       owners:
-      - https://raw.githubusercontent.com/kubernetes/perf-tests/master/OWNERS
+      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/kubemark/OWNERS
+      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/test/kubemark/OWNERS
```
[the remark this answers is not in the document]
Comment (wojtek-t) [reply sigs.yaml]: done

File: sigs.yaml (hunk 3 above)
Comment (shyamjvs) [reply sigs.yaml]: Yeah, that sgtm.

**Comment by wojtek-t** [comment]
@cblecker @thockin - can one of you approve that?

**Pull request merged**